communities/hacs/readme.md

```diff
@@ -0,0 +1,38 @@
+[![Build Status](https://travis-ci.org/HaCSBCU/hacsbcu.github.io.svg?branch=master)](https://travis-ci.org/HaCSBCU/hacsbcu.github.io)
+
+![Website Screenshot](http://i.imgur.com/lJnqm1B.jpg)
+
+# HaCS
+This is the website for HaCS - Hackathon and Computing Society at Birmingham City University. This website acts as our central resource for content and as a playground for our members to try out new ideas and work together with others.
+
+We're open to contributions and are always improving our site. If there are any suggestions you might have or something you'd like to work on yourself feel free to open an [issue](https://github.com/HaCSBCU/hacsbcu.github.io/issues/new).
+
+If you are a BCU student and would like to contribute to this site but don't know how yet, it might be a good idea to drop us a message on our [Facebook group](https://www.facebook.com/groups/hacsbcu/) or drop into one of our workshops.
+
+
+## Building
+
+We're using jekyll to build the site. If you want to test locally you will need [ruby](https://rvm.io/rvm/install) and [jekyll](https://jekyllrb.com/) to be installed.
+
+Once you have ruby installed:
+
+`$ gem install bundler`
```

Comment:
Something I've come across when introducing beginners to projects is that the `$` at the start of a command can be more of a hindrance than a help — if people try to copy and paste the command into their terminal and don't know they have to remove the `$` it won't work.